### PR TITLE
Update Bundle Install manual page for Bundler 3

### PR DESCRIPTION
**Pull request by colby-swandale**
## Overview

This PR rewrites the `bundle install` manual page for Bundler 2. I've removed documentation for the deprecated options, updated existing chapters and also added chapters to address questions people may have about moved functionality (ie: new config options).

### CODE REVIEW

**Comment by colby-swandale** [comment]
@rubymorillo i would love to have your opinion on this as well!

File: man/bundle-install.ronn [hunk 1]
```diff
@@ -99,33 +54,16 @@ time `bundle install` is run, use `bundle config` (see bundle-config(1)).
...
 * `--retry=[<number>]`:
   Retry failed network or git requests for <number> times.
 
-* `--shebang=<ruby-executable>`:
-  Uses the specified ruby executable (usually `ruby`) to execute the scripts
-  created with `--binstubs`. In addition, if you use `--binstubs` together with
-  `--shebang jruby` these executables will be changed to execute `jruby`
-  instead.
+* `--redownload`:
+  Force download every gem, even if the required versions are already available
```
Comment (rubymorillo) [inline man/bundle-install.ronn]: Force downloads every gem (add 's' after 'download'), otherwise looks good to me!

File: man/bundle-install.ronn [hunk 2]
```diff
@@ -135,72 +73,17 @@ time `bundle install` is run, use `bundle config` (see bundle-config(1)).
...
-   read them.
-
-   As a result, `bundle install --deployment` installs gems to
-   the `vendor/bundle` directory in the application. This may be
-   overridden using the `--path` option.
-
 ## SUDO USAGE
 
-By default, Bundler installs gems to the same location as `gem install`.
+By default, Bundler installs gems into `.bundle`.
```
Comment (rubymorillo) [inline man/bundle-install.ronn]: 👍

File: man/bundle-install.ronn [hunk 3]
```diff
@@ -135,72 +73,17 @@ time `bundle install` is run, use `bundle config` (see bundle-config(1)).
...
 
-In some cases, that location may not be writable by your Unix user. In
+In some cases, that location may not be writable by a unprivileged user. In
```
Comment (rubymorillo) [inline man/bundle-install.ronn]: "An unprivileged" user (missing 'n'). otherwise lgtm

File: man/bundle-install.ronn [hunk 4]
```diff
@@ -377,6 +264,24 @@ should first try to run `bundle install`, which will guarantee that no
 other gem in the Gemfile(5) is impacted by the change. If that
 does not work, run [bundle update(1)][bundle-update(1)].
 
+## Transitioning from Bundler 1
+
+Bundler 2 has removed several options from the CLI that are configurable
+through the [bundle config(1))[bundle-config] command.
```
Comment (rubymorillo) [inline man/bundle-install.ronn]: > [bundle config(1))[bundle-config] command.

looks like a typo there (extra parenthesis, or missing bracket)

File: man/bundle-install.ronn [hunk 5]
```diff
@@ -228,26 +111,29 @@ gems into their final location.
...
+While the `without` config will skip _installing_ the gems in the
 specified groups, it will still _download_ those gems and use them to
 resolve the dependencies of every gem in your Gemfile(5).
 
 This is so that installing a different set of groups on another
  machine (such as a production server) will not change the
 gems and versions that you have already developed and tested against.
 
-`Bundler offers a rock-solid guarantee that the third-party
+Bundler offers a rock-solid guarantee that the third-party
```
Comment (rubymorillo) [inline man/bundle-install.ronn]: > +Bundler offers a rock-solid guarantee that the third-party

Not a big deal, but I'd remove "rock-solid". The guarantee should be enough :) (and some folks might see that and think _eye roll_)

File: man/bundle-install.ronn [hunk 6]
```diff
@@ -135,72 +73,17 @@ time `bundle install` is run, use `bundle config` (see bundle-config(1)).
   in the manner required. Using this option implicitly sets `path`, which is a
   [remembered option][REMEMBERED OPTIONS].
 
-* `--system`:
-  Installs the gems specified in the bundle to the system's Rubygems location.
-  This overrides any previous configuration of `--path`.
-
 * `--trust-policy=[<policy>]`:
   Apply the Rubygems security policy <policy>, where policy is one of
```
Comment (rubymorillo) [inline man/bundle-install.ronn]: > Apply the Rubygems security policy <policy>, 

"Applies"

File: man/bundle-install.ronn [hunk 7]
```diff
@@ -99,33 +54,16 @@ time `bundle install` is run, use `bundle config` (see bundle-config(1)).
...
-  setting. Bundler shares this location with Rubygems, `gem install ...` will
-  have gem installed there, too. Therefore, gems installed without a
-  `--path ...` setting will show up by calling `gem list`. Accordingly, gems
-  installed to other locations will not get listed.
-
 * `--quiet`:
   Do not print progress information to the standard output. Instead, Bundler
   will exit using a status code (`$?`).
 
 * `--retry=[<number>]`:
```
Comment (rubymorillo) [inline man/bundle-install.ronn]: > Retry failed network or git requests for <number> times.

"Retries"

File: man/bundle-install.ronn [hunk 8]
```diff
@@ -260,13 +146,14 @@ For a simple illustration, consider the following Gemfile(5):
...
 In this case, `sinatra` depends on any version of Rack (`>= 1.0`), while
-`rack-perftools-profiler` depends on 1.x (`~> 1.0`).
+`rack-perftools-profiler` depends on Rack 1.x (`~> 1.0`).
 
-When you run `bundle install --without production` in development, we
-look at the dependencies of `rack-perftools-profiler` as well. That way,
-you do not spend all your time developing against Rack 2.0, using new
-APIs unavailable in Rack 1.x, only to have Bundler switch to Rack 1.2
-when the `production` group _is_ used.
+When you configure bundler to exclude `production` with
```
Comment (rubymorillo) [inline man/bundle-install.ronn]: > When you configure bundler to exclude `production` with

capitalize "Bundler"

**Comment by oniofchaos** [comment]
Looks like just a merge conflict now :trollface:

**Comment by deivid-rodriguez** [comment]
@colby-swandale I think this PR is really good to get our documentation ready for bundler 3. Do you mind if I reopen it and set it as blocked? Or would you prefer that I open a separate PR so you don't get bothered with notifications (I guess that's why you closed it?).

Just let me know :)

**Pull request closed without merging**